### DISCUSSION TITLE
`up-alias` with multiple alternatives

### DISCUSSION THREAD

**Discussion by alexandremjacques** [unpoly/unpoly#429] [Q&A] [answered]
Hi, 

I'm trying to make `up-alias` work with 2 alternatives like bellow:
 
```
<div class="collapse navbar-collapse" id="navbarSupportedContent">
            <ul class="navbar-nav me-auto mb-2 mb-lg-0" up-nav>
                <li class="nav-item me-3" up-alias="/dashboard /tasks">
                    <a class="nav-link d-flex align-items-center text-muted"
                       href="{% url 'dashboard' %}"
```

It works fine finding `/dashboard` but not `/tasks` even though the URL is a simple `http://localhost:8080/tasks`.

Is there anything I'm missing?

TIA

**Comment by triskweline** [comment, marked as answer]
Your code looks good to me.

If you can prepare a [Glitch project](https://glitch.com/edit/#!/empty-unpoly-project) demonstrating the issue we may be able to debug this further.

**Reply by alexandremjacques** [reply]
I'll have to investigate further. Apparently it works fine outside my app. 

Sorry about that. Should have tried it first.
